test/S3Test.js

```diff
@@ -1,285 +1,343 @@
 var should = require('should'),
-  crypto = require('crypto'),
-  os = require('os'),
-  fs = require('fs'),
-  path = require('path'),
-  request = require('request'),
-  async = require('async'),
-  S3 = require('../lib/S3'),
-  sinon = require('sinon'),
-  proxyquire = require('proxyquire')
-
-var bucket =  process.env.CRATE_BUCKET
+        crypto = require('crypto'),
+        os = require('os'),
+        fs = require('fs'),
+        path = require('path'),
+        request = require('request'),
+        async = require('async'),
+        S3 = require('../lib/S3'),
+        sinon = require('sinon'),
+        proxyquire = require('proxyquire')
+
+var bucket = process.env.CRATE_BUCKET
 var key = process.env.CRATE_KEY
 var secret = process.env.CRATE_SECRET
 var region = process.env.CRATE_REGION
 
-describe('S3', function() {
-
-  it('should require options', function(done) {
-    (function() {
-      new S3()
-    }).should.throw()
-
-    done()
-  })
-
-  it('should require a key', function(done) {
-    (function() {
-      new S3({})
-    }).should.throw()
-
-    done()
-  })
-
-  it('should require a secret', function(done) {
-    (function() {
-      new S3({
-        key: 'foo'
-      })
-    }).should.throw()
-
-    done()
-  })
-
-  it('should require a bucket', function(done) {
-    (function() {
-      new S3({
-        key: 'foo',
-        secret: 'bar'
-      })
-    }).should.throw()
-
-    done()
-  })
-
-  it('should set a default acl', function(done) {
-    var s3 = new S3({
-      key: 'foo',
-      secret: 'bar',
-      bucket: 'baz'
+describe('S3', function () {
+
+    it('should require options', function (done) {
```

Comment:
Again, all the two-space indentation seems to have changed to four-spaces.  I'll probably run [standard](https://github.com/feross/standard) in a pre-commit hook in future to catch this stuff.